src/lib/Scenes/Artwork/Components/ImageCarousel/useGlobalState.ts

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useMemo, useRef, useState } from "react"
+
+function useForceUpdate() {
+  const setEpoch = useState(0)[1]
+  const isMounted = useRef(false)
+  useEffect(() => {
+    isMounted.current = true
+    return () => {
+      isMounted.current = false
+    }
+  })
+  return useMemo(() => () => isMounted.current && setEpoch(x => x + 1), [])
+}
+
+export class GlobalState<T> {
```

Comment:
We could add a short description here, maybe even a link back to this PR, for context.

Reply:
great idea! done